app/lib/test_applications.rb

```diff
@@ -111,7 +115,8 @@ def create_application(states:, courses_to_apply_to: nil, apply_again: false, co
         # To prevent immediate sending to provider, which uses the current
         # time to decide whether edit_by has passed, temporarily set edit_by to
         # a date in the future relative to now
-        @application_form.update_columns(submitted_at: time, edit_by: Time.zone.now + 1.day)
+        @application_form.application_choices.each { |choice| choice.update_columns(updated_at: time) }
+        @application_form.update_columns(submitted_at: time, edit_by: Time.zone.now + 1.day, updated_at: time)
```

Comment:
From call: we could push this "real" edit_by date further out so the application remained in `application_complete` for a few days, which is a bit more realistic